stdlib/2and3/codecs.pyi

```diff
@@ -4,6 +4,7 @@ import sys
 from typing import (
     BinaryIO,
     Callable,
+    Generator,
     IO,
     Iterable,
     Iterator,
```

Comment:
Is this import still necessary?

Reply:
I assume you're asking if Iterator is still required. Seem's you're right that it isn't but my linter didn't pick it up. Submitting another commit to remove.